README.md

```diff
@@ -1,5 +1,17 @@
 <a name="readme-top"></a>
 
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
 # 📗 Table of Contents
 
```

Comment:
- [ ] The key features and future features need to reflect the project. Also, fix the broken MIT license and issue links